src/main/java/mate/academy/spring/mapper/impl/request/OrderRequestMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.mapper.impl.request;
+
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.request.OrderRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderRequestMapper implements DtoRequestMapper<OrderRequestDto, Order> {
```

Comment:
orderrequestdto and its mapper aren't used
![image](https://user-images.githubusercontent.com/49767049/233399687-8d96875c-ed10-4aa1-b6cc-0ce3992e57fd.png)